composer.json

```diff
@@ -6,13 +6,16 @@
     "license": "MIT",
     "minimum-stability": "dev",
     "require": {
-        "php": "^7.2",
-        "itk-dev/pretix-api-client-php": "^1.0",
+	    "php": "^8.1",
         "nicoeg/dawa": "^1.0"
     },
     "require-dev": {
         "drupal/coder": "^8.3.1",
-        "dealerdirect/phpcodesniffer-composer-installer": "dev-master"
+        "dealerdirect/phpcodesniffer-composer-installer": "^1.0",
+        "phpstan/extension-installer": "^1.2",
+        "mglaman/phpstan-drupal": "^1.1",
+        "phpstan/phpstan-deprecation-rules": "^1.1",
+        "itk-dev/pretix-api-client-php": "^1.0"
```

Comment:
`itk-dev/pretix-api-client-php` must not be a `dev` requirement. The version should be updated to `^1.1` to make sure that we get the PHP 8.1 version.

Reply:
Fixed